Flickr/Sources/FlickrAppVCs/FavoritesCoordinator.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  FavoritesCoordinator.swift
+//  Flickr
+//
+//  Created by Ananth Desai on 20/03/22.
+//
+
+import Foundation
+import UIKit
+
+class FavoritesCoordinator {
+    func returnRootNavigator() -> UINavigationController {
+        let favoritesVC = FavoritesVC()
+        favoritesVC.title = favoritesVcTitle
+        let rootNav = UINavigationController(rootViewController: favoritesVC)
+        if #available(iOS 13.0, *) {
+            let navbarAppearance = UINavigationBarAppearance()
+            navbarAppearance.backgroundColor = navigationBarBackgroundColor
+            rootNav.navigationBar.standardAppearance = navbarAppearance
+            rootNav.navigationBar.scrollEdgeAppearance = navbarAppearance
```

Comment:
Why are you assigning empty `UINavigationBarAppearance`

Reply:
No reply?

Reply:
As mentioned earlier, I had added this for the background of the navigation bar.
Setting the background color of the navigation bar didn't work for the part of screen above the notch. UINavigationBarAppearance solves the issue.